meson.build

```diff
@@ -8,12 +8,19 @@ rtdl_include_dirs = [
 ]
 libc_include_dirs = [
 	include_directories('options/internal/include'),
-	include_directories('options/internal/x86_64-include'),
 	include_directories('options/elf/include'),
 	include_directories('options/lsb/include'),
 	include_directories('options/glibc/include'),
-
 ]
+
+if host_machine.cpu_family() == 'x86_64'
+	libc_include_dirs += include_directories('options/internal/x86_64-include')
+elif host_machine.cpu_family() == 'aarch64'
+	libc_include_dirs += include_directories('options/internal/aarch64-include')
+else
+	error('Unknown architecture')
+endif
```

Comment:
If statements can be replaced with
```
include_directories('options/internal' / host_machine.cpu_family() + '-include')
```
so adding more architectures won't require additional changes